packages/apsara-ui/src/TableV2/VirtualisedTable.tsx

```diff
@@ -28,6 +29,19 @@ interface ITableProps {
     loading?: boolean;
 }
 
+type RenderFunction<T, U = T> = (props: { row: { original: U } }) => any;
```

Comment:
can we investigate if we really need to nest the value inside `original`?
if possible let's not do that.